src/components/ScrollableTabView/ScrollableTabView.js

```diff
@@ -78,10 +79,10 @@ class ScrollableTabView extends React.Component {
 
   scrollToTop = () => {
     const { scrollRef, scrollHeight, isHeaderFolded } = this.props
-
+    const scrollNode = getSafelyScrollNode(scrollRef)
     return (
       isHeaderFolded &&
-      scrollRef.scrollTo({
+      scrollNode.scrollTo({
```

Comment:
below there is also check for scrollNode, shouldn't it be here also? or maybe that below is unnecessary

Reply:
Yeah, let's double check that scrollNode :)